src/core/namespace.h

```diff
@@ -150,6 +159,7 @@ struct NamespaceParameters {
         ProtectSystem protect_system;
         ProtectProc protect_proc;
         ProcSubset proc_subset;
+        PrivateTmp private_tmp_as_tmpfs;
```

Comment:
the "_as_tmpfs" suffix doesn't make much sense now. just call this `private_tmp`?